styles.css

```diff
@@ -8,14 +8,13 @@
 
 body {
```

Comment:
- [ ] It is essential for a website to be responsive and scale appropriately with respect to screen size. When I try to reduce the screen, the content of your webpage is not scaling at all, therefore introducing some horizontal scrollbars, while some contents are overlapping each other which is not a good user experience. 

To make your website responsive:
1. Use percentages (%) for images
2. Use rem/em for fonts
3. Make appropriate use of flexbox/grid 👍🏽 

e.g 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/194629492-ac97a769-cc78-44a9-9e5a-75c9c0372ae8.png)